### ISSUE TITLE
Deprecation Warning: 'localtuya' Integration Accessing Deprecated hass.helpers.service Method

### ISSUE THREAD

**Issue by 0Styless** [rospogrigio/localtuya#1813]
## The problem

Home-Assistant deprecation warning.

## Environment

- Localtuya version: v5.2.2
- Home Assistant Core version: 2024.10.1
- [x] Does the device work using the Home Assistant Tuya Cloud component ?
- [x] Does the device work using the Tinytuya (https://github.com/jasonacox/tinytuya) command line tool ?
- [x] Was the device working with earlier versions of localtuya ? Which one?
- [x] Are you using the Tuya/SmartLife App in parallel ?

## Provide Home Assistant traceback/logs

```
WARNING (MainThread) [homeassistant.helpers.frame] Detected that custom integration 'localtuya' accesses hass.helpers.service. This is deprecated and will stop working in Home Assistant 2024.11, it should be updated to import functions used from service directly at custom_components/localtuya/__init__.py, line 165: hass.helpers.service.async_register_admin_service
```

**Comment by fl0rian089** [comment]
i think they updated the deprecation version

Detected that custom integration 'localtuya' accesses hass.helpers.service. This is deprecated and will stop working in Home Assistant 2025.5, it should be updated to import functions used from service directly at custom_components/localtuya/__init__.py, line 165: hass.helpers.service.async_register_admin_service(, please create a bug report at https://github.com/rospogrigio/localtuya/issues

**Comment by braetkuegeli** [comment]
@fl0rian089
yes, according [changelog](https://www.home-assistant.io/changelogs/core-2024.11) period is exteded for 6 months:

`Extend deprecation period for hass.helpers by 6 months ([@jpbede](https://github.com/jpbede) - [#128038](https://github.com/home-assistant/core/pull/128038))`

**Comment by fl0rian089** [comment]
@braetkuegeli 

i read the system log faster than the changelog. thanks! :-)

**Comment by CloCkWeRX** [comment]
Same as #1800
